components/sst_importer/src/import_file.rs

```diff
@@ -458,20 +458,33 @@ impl ImportDir {
 }
 
 const SST_SUFFIX: &str = ".sst";
-
+// version 2: compared to version 1 which is the default version, we will check
+// epoch of request and local region in write API.
+pub const API_VERSION_2: i32 = 2;
+
+/// sst_meta_to_path will encode the filepath with default api version (current
+/// is 2). So when the SstMeta is created in old version of TiKV and filepath
+/// will not correspond to the real file, in the deletion logic we can't remove
+/// these files.
 pub fn sst_meta_to_path(meta: &SstMeta) -> Result<PathBuf> {
     Ok(PathBuf::from(format!(
-        "{}_{}_{}_{}_{}{}",
+        "{}_{}_{}_{}_{}_{}{}",
```

Comment:
Does it mean this SST cannot be parsed by older version. 
It's fine to me, but need to document somewhere later that it's not compatible with older version.

Reply:
In the old version of `parse_meta_from_path`, it will only use the first 5 fields after split the string with `_`, so the new version should be able to be ignored.